docs/spec/spec.md

```diff
@@ -9,10 +9,90 @@ These are Kubernetes resources that been introduced using Custom Resource
 
+## kind: Broker
+
+### group: eventing.knative.dev/v1alpha1
+
+_A Broker represents an event mesh. It logically receives events on its input
+domain and forwards them to subscribers defined by one or more matching
+Trigger._
+
+### Object Schema
+
+#### Spec
+
+| Field           | Type        | Description                                                                                                        | Constraints                                      |
+| --------------- | ----------- | ------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------ |
+| channelTemplate | ChannelSpec | The template used to create Channels internal to the Broker. Defaults to to the default Channel for the namespace. | Only Provisioner and Arguments may be specified. |
+
+#### Status
+
+| Field              | Type        | Description                                                                                  | Constraints |
+| ------------------ | ----------- | -------------------------------------------------------------------------------------------- | ----------- |
+| observedGeneration | int64       | The 'Generation' of the Broker that was last processed by the controller.                    |             |
+| address            | Addressable | Address of the endpoint which meets the [_Addressable_ contract](interfaces.md#addressable). |             |
+| conditions         | Conditions  | Broker conditions.                                                                           |             |
+
+##### Conditions
+
+- **Ready.** True when the Broker is provisioned and ready to accept events.
+- **Addressable.** True when the Broker has an resolved address in it's status.
+
+#### Events
+
+- BrokerReconciled
+- BrokerUpdateStatusFailed
+
+---
+
+## kind: Trigger
+
+### group: eventing.knative.dev/v1alpha1
+
+_A Trigger represents a subscriber of events with a filter for a specific
+broker._
+
+### Object Schema
+
+#### Spec
+
+| Field        | Type           | Description                                                                                                                                                                | Constraints |
+| ------------ | -------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| broker       | String         | Broker is the broker that this trigger receives events from. Defaults to 'default'.                                                                                        |             |
+| filter       | TriggerFilter  | Filter is the filter to apply against all events from the Broker. Only events that pass this filter will be sent to the Subscriber. Defaults to subscribing to all events. |             |
```

Comment:
Should `filter` be required (even if it is empty)?

Reply:
I don't think so but I could be convinced. What do you think @grantr ?